Sharpmake/Project.cs

```diff
@@ -254,6 +254,13 @@ public uint DependenciesOrder
             set { SetProperty(ref _dependenciesOrder, value); }
         }
 
+        private bool _autoDependenciesOrder = false;
```

Comment:
I wonder if this shouldn't rather be on the conf itself, so it would allow conditionally activating it on platforms where it matter, like linux

Reply:
Then I'd put it in the conf, it allows greater flexibility